api/v1beta1/verticadb_types.go

```diff
@@ -291,6 +291,23 @@ type LocalObjectReference struct {
 	Name string `json:"name,omitempty" protobuf:"bytes,1,opt,name=name"`
 }
 
+// SubclusterSelection is used to select between existing subcluster by name
+// or provide a template for a new subcluster.  This is used to specify what
+// subcluster gets client routing for subcluster we are restarting during online
+// image change.
+type SubclusterSelection struct {
+	// +operator-sdk:csv:customresourcedefinitions:type=spec
+	// +kubebuilder:validation:Optional
+	// Names of existing subclusters to use for temporary routing of client
+	// connections.  The operator will use the first subcluster that is online.
+	Names []string `json:"names,omitempty"`
+
+	// +operator-sdk:csv:customresourcedefinitions:type=spec
+	// +kubebuilder:validation:Optional
+	// A new subcluster will be created using this as a template.
+	Template Subcluster `json:"template,omitempty"`
```

Comment:
Will we always use a single subcluster as temporary?

Reply:
Yes, if we use the template a single subcluster is created.  If we reuse an existing subcluster 1 or 2 subclusters can be specified -- the 2nd subcluster is only used when we are upgrading the 1st.

I'll add more comments here.